src/components/Dropdown/V2/__tests__/Dropdown.Item.test.tsx

```diff
@@ -226,6 +298,43 @@ describe('renderItem', () => {
 
     expect(el.length).toBeTruthy()
   })
+
+  test('Can render custom markup for item when multiselect enabled', () => {
+    const CustomItem = (props = {}) => {
+      // @ts-ignore
+      return <div className="ron">{props.label}</div>
+    }
+
+    const wrapper = mount(
+      <Item
+        label="Champ"
+        renderItem={CustomItem}
+        getState={() => {
+          return {
+            allowMultipleSelection: true,
+          }
+        }}
+      />
+    )
+
+    const el = wrapper.find('div.ron')
+
+    expect(el.length).toBeTruthy()
+  })
+
+  test('Renders default markup (itemSelectedCheck) for item when multiselect enabled', () => {
+    const wrapper = shallow(<Item label="Champ" value="hello" />)
```

Comment:
@tinkertrain Curious! How come this one is shallow mounted? :)

Reply:
no reason! I was trying something and this is a leftover